batch/batch/front_end/front_end.py

```diff
@@ -611,8 +667,16 @@ async def get_job_container_log(request, batch_id):
     containers = job_tasks_from_spec(record)
     if container not in containers:
         raise web.HTTPBadRequest(reason=f'unknown container {container}')
-    job_log = await _get_job_container_log(app, batch_id, job_id, container, record)
-    return web.Response(body=job_log)
+    container_log = await _get_job_container_log(app, batch_id, job_id, container, record)
+
+    resp = web.StreamResponse()
+    await resp.prepare(request)
+    if container_log is not None:
+        async with container_log:
+            while b := await container_log.read(1024):
```

Comment:
I feel like 1KiB is a bit small, but we can fiddle with these parameters in future PRs.